packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -248,6 +249,7 @@ const BaseLink = ({
       motionDuration={motionDuration}
       motionEasing={motionEasing}
       setCurrentInteraction={setCurrentInteraction}
+      {...props}
```

Comment:
@anuraghazra ^ We shouldn't do this. It will allow others to override styles of Link and create design inconsistencies. It could also lead to passing of dangerous attributes like `as`

Reply:
Done.